tools/clang/test/CodeGenHLSL/quick-test/check_status.hlsl

```diff
@@ -0,0 +1,15 @@
+// RUN: %dxc -E main -T cs_6_0 %s | FileCheck %s
```

Comment:
Add a test for:

1. user inserted checkAccessFullyMapped
2. get status but do not read it.